aleph-client/rustfmt.toml

```diff
@@ -0,0 +1,7 @@
+edition = "2021"
+use_field_init_shorthand = true
+reorder_modules = true
+
+imports_granularity = "Crate"
+group_imports = "StdExternalCrate"
+reorder_imports = true
```

Comment:
Does running it change anything? If so, can we add it and run on the whole project in a separate PR?

Reply:
No, this is a bit of a quirk I did here. That is the same rustfmt toml as this one https://github.com/Cardinal-Cryptography/aleph-node/blob/main/rustfmt.toml , I've copied it since I wanted to build context to be in `aleph-client` directory. In that case docker prevents using `COPY` command from the parent directory, when the original `rustmft.toml` file originates. A symbolic link does not work either, so it must be a hard copy. I know it's a duplication.